modules/libevent/2.1.12-stable.bcr.0/overlay/event2-config_apple.h

```diff
@@ -0,0 +1,520 @@
+/* event-config_apple.h
+ *
+ * This file was generated by cmake but modified for Bazel.
```

Comment:
Could you add a comment explaining these modifications for Bazel? This could help update this file in future versions.

Reply:
I actually left an entire `README.md` with maintainer notes, so I added a note that it exists in these files.